test/config/integration/google_com_proxy_port_0_with_kill_request_filter.yaml

```diff
@@ -0,0 +1,49 @@
+admin:
+  access_log_path: "{{ null_device_path }}"
+  address:
+    socket_address:
+      address: "{{ ip_any_address }}"
+      port_value: 0
+
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+      socket_address:
+        address: "{{ ip_any_address }}"
+        port_value: 0
+    filter_chains:
+    - filters:
+      - name: http
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          stat_prefix: ingress_http
+          codec_type: AUTO
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: local_service
+              domains: ["*"]
+              routes:
+              - match: { prefix: "/" }
+                route: { host_rewrite_literal: www.google.com, cluster: service_google }
+          http_filters:
+          - name: envoy.filters.http.kill_request
+            typed_config:
```

Comment:
Why do you need a separate integration test config? Can't you just use helpers in https://github.com/envoyproxy/envoy/blob/master/test/config/utility.h to inject the filter on top of an existing config?

Reply:
I see.. you want to use it directly in `main_common_test.cc`. I suggest dramatically minimizing this file and scoping the filegroup to just this one test.

Reply:
Or place in `test/exe/testdata`.

Reply:
Thanks Harvey. Moved this file to `test/exe/testdata` and also made it simplified